timesketch/lib/analyzers/interface.py

```diff
@@ -324,6 +325,34 @@ def add_aggregation(
         db_session.commit()
         return aggregation
 
+    def add_aggregation_group(self, name, description='', view_id=None):
+        """Add aggregation Group to the sketch.
+
+        Args:
+            name: the name of the aggregation run.
+            description: description of the aggregation, visible in the UI,
```

Comment:
nit: Optional description of the aggregation (visible in the UI)

Reply:
ack